javascript/javascript1/week1/task-3.js

```diff
@@ -0,0 +1,15 @@
+
+// Housey pricey (A house price estimator)
+ 
+let houseWidth = prompt('house width?');
+let houseDeep = prompt('house deep?');
+let houseHigh = prompt('hose higt?');
+const volumeInMeters = houseDeep * houseHigh * houseWidth;
+let gardenSizeInM2 = prompt('garden size in m2?');
+let housePrice = prompt('house price?');
+
+if (housePrice >= volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300) {
+   console.log("You pay too much!");
+} else {
+console.log('It is a good price =)');
+};
```

Comment:
good using for prompt method, but it's better to follow the homework requirements  
always any user inputs need validation and check type, and here in your case, the first condition will never be true if the user adds strings in the input field..,

Reply:
Thank you! I get some ideas, maybe I have to use objects instead of it?